java-checks-test-sources/default/src/main/java/checks/spring/StaticFieldInjectionNotSupportedCheckSample.java

```diff
@@ -0,0 +1,91 @@
+package checks.spring;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Target;
+import org.springframework.stereotype.Component;
+import org.springframework.beans.factory.annotation.Autowired;
+import jakarta.inject.Inject;
+import org.springframework.beans.factory.annotation.Value;
+
+public class StaticFieldInjectionNotSupportedCheckSample {
+
+  @Target({ElementType.FIELD})
+  @interface AnAnnotation {
+
+  }
+
+  @Target({ElementType.FIELD})
+  @Autowired
+  @interface MyAutowired {}
+
+  // we should only raise if the DI framework is spring
```

Comment:
Is this something we should check by looking at the annotation below? We are currently ignoring the context